util/req/req.go

```diff
@@ -0,0 +1,3 @@
+// package req provides request body validation facilities. Using struct tags
+// you may define rules in order to validate structured user input.
+package req
```

Comment:
Why is this file empty ?

Reply:
I didn’t know in which file to put the package documentation, so I put it here, in a file eponymous to the package, which will not contain code.